modules/test/playwright/helpers/ObjectApiHelper.ts

```diff
@@ -37,9 +38,9 @@ export class ObjectApiHelper {
 		};
 
 		return this.apiHelpers.postResponse(
-			`${this.apiHelpers.baseUrl}c/${
-				applicationName.toLowerCase() + 's'
-			}`,
+			`${this.apiHelpers.baseUrl}c/${formatPlural(
```

Comment:
I  just checked that we could get the path from the `restContextPath` field of the object definition:

```
curl -X GET -u test@liferay.com:test http://localhost:8080/o/object-admin/v1.0/object-definitions
```

```
{
    "accountEntryRestricted" : false,
    "accountEntryRestrictedObjectFieldName" : "",
    [...],
    "active" : true,
    "dateCreated" : "2024-01-22T10:11:23Z",
    "dateModified" : "2024-01-22T10:11:36Z",
    "defaultLanguageId" : "en_US",
    "enableCategorization" : true,
    "enableComments" : false,
    "enableLocalization" : false,
    "enableObjectEntryDraft" : false,
    "enableObjectEntryHistory" : false,
    "externalReferenceCode" : "6790f1a2-1e3a-6503-be37-899169f21487",
    "id" : 114129,
    "label" : {
      "en_US" : "Test"
    },
    "modifiable" : true,
    "name" : "Test",
    "objectActions" : [ ],
    "objectFields" : [ {
       [...],
    } ],
    "objectLayouts" : [ ],
    "objectRelationships" : [ ],
    "objectValidationRules" : [ ],
    "objectViews" : [ ],
    "panelCategoryKey" : "",
    "parameterRequired" : false,
    "pluralLabel" : {
      "en_US" : "PluralTest"
    },
    "portlet" : true,
    "restContextPath" : "/o/c/tests",
    "scope" : "company",
    "status" : {
      "code" : 0,
      "label" : "approved",
      "label_i18n" : "Approved"
    },
    "system" : false,
    "titleObjectFieldName" : "id"
  }
```

Reply:
That could apply to our integration tests too